samples/debug/ppi_trace/sysbuild/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if (BOARD MATCHES "nrf52")
+  set(SNIPPET bt-ll-sw-split ${SNIPPET} CACHE STRING "" FORCE)
+endif()
```

Comment:
since this is just for the main application, this should be in sysbuild.cmake and use the proper Kconfig symbols for the SoCs